src/sections/Meshery/index.js

```diff
@@ -38,11 +39,15 @@ const MesheryPage = () => {
                 <FiDownloadCloud size={21} className="button-icon" />
               </Button>
             </Col>
-            <Col lg={6} md={6} className="meshery-hero-img">
-              <video autoPlay muted loop controls className="meshery-video">
+            <Col lg={6} md={6} className="meshery-hero-img desc-text">
+              <video autoPlay muted loop controls className="meshery-video" >
```

Comment:
```suggestion
            <Col lg={6} md={6} className="meshery-hero-img">
              <video autoPlay muted loop controls className="meshery-video">
```

Reply:
If I remove the css `desc-text` then the button won't look like the other two button.